camel-core/src/main/java/org/apache/camel/impl/verifier/DefaultComponentVerifier.java

```diff
@@ -123,7 +123,7 @@ protected void verifyParametersAgainstCatalog(ResultBuilder builder, Map<String,
             stream(result.getInvalidEnum())
                 .map(entry ->
                     ResultErrorBuilder.withIllegalOption(entry.getKey(), entry.getValue())
-                        .attribute("enum.values", result.getEnumChoices(entry.getKey()))
+                        .detail(ResultErrorBuilder.errorAttribute("enum.values"), result.getEnumChoices(entry.getKey()))
```

Comment:
It's because I think of the 'dedail' == 'attribute'  + value So this method only created the key, not the full detail.

Maybe only 'attribute' ? (as 'error' can be already inferred from the context / class)